### ISSUE TITLE
[You Can Ignore It, But Please Dont][Sorry To Ask this if this is A Dumb Question] - Passcode for Iphone XR, X

### ISSUE THREAD

**Issue by alqudri99** [checkra1n/BugTracker#2279]
hy, i will keep this very short question. can i do the jailbreak with passcode that was setup before the jailbreak (in case i forgot the password)?. than if the answer is no, are the answer is absolutely no or it might be have a chance but not tested yet?

and, base on this sentences. what if the device is A11 device but ios version is below 14 ? are this device possible to jailbreak without unlock the device?
<img width="807" alt="Screen Shot 2022-05-12 at 01 25 44" src="https://user-images.githubusercontent.com/69712318/167920188-ad0fc33c-3505-4daa-924b-2d7a21b7e60a.png">

Disclaimer : i do read the changelog, but just need to validate the probability and test my luck on it. Thanks!

**Comment by worf1337** [comment]
if you forgot the passcode checkra1n can't remove it...

**Comment by alqudri99** [comment]
> if you forgot the passcode checkra1n can't remove it...

no, i dont ask if checkra1n. what i am asking is, can checkra1n jailbreak A11 in password mode, that what i am asking. Thanks
